WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -717,6 +716,19 @@ extension ProductFormViewController: KeyboardScrollable {
     }
 }
 
+// MARK: - Navigation actions handling
+//
+private extension ProductFormViewController {
```

Comment:
I did not see this being called when navigating back from a product with unsaved changes. Is that expected?

Reply:
@Ecarrion did you test on an iOS 13.4+ simulator/device? there's an unfortunate bug from Apple that Paolo looked into in https://github.com/woocommerce/woocommerce-ios/pull/2110. if you tried on an iOS 12 simulator or a TestFlight build, the discard changes alert is expected to show up 🙂

Reply:
Oh yeah, I did test it in iOS 13.6.
Thanks for the heads up about that issue :-)